docs/en/02_Developer_Guides/12_Search/02_FulltextSearch.md

```diff
@@ -71,11 +71,14 @@ Example DataObject:
 Performing the search:
 
 	:::php
-	SearchableDataObject::get()->filter('SearchFields:fulltext', 'search term');
```

Comment:
The fact that even our docs haven't cared about casing should give you an indication of how widespread this problem will be